openlane_pr_tag/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Create a new pull request to update OpenLane tag.
+
+inputs:
+  ol_tag:
+    description: OpenLane tag to use in PR
+    required: true
+  path:
```

Comment:
does this actually change?

Reply:
It's a choice for the user. We could also make a fixed value.

Reply:
I assume this will more or less always be used in combination with `openlane_run` to update the tool to the corresponding version that was succesfully tested?

Did you consider merging the two action together, so that the end-users don't have to wire them manually together?

Reply:
Merged with `openlane_run` and added an `bool` option to control PR creation.